### PR TITLE
Update Integration QSG doc with new changes and screenshots

### PR DESCRIPTION
**Pull request by arunans23**
$subject

### CODE REVIEW

File: en/docs/get-started/quick-start-guide/integration-qsg.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-1. Go to the [website](https://wso2.com/api-management/) to download WSO2 API Manager and WSO2 Integration Studio. 
+1. Go to the [website](https://wso2.com/api-management/) to download WSO2 Micro Integrator and WSO2 Integration Studio. 
 
     When you click **Download**, the installation options will be listed. For this quick start, you can either download and run the **installer**, or use the **binary** file.
 
     !!! Info
         For more information, see the [installation instructions]({{base_path}}/install-and-setup/install-and-setup-overview/#installing_1).
 
-2. Download the [sample files](https://github.com/wso2/docs-ei/blob/master/en/micro-integrator/docs/assets/attach/quick-start-guide/MI_QSG_HOME-JDK11.zip). From this point onwards, let's refer to this folder as `<MI_QSG_HOME>`.
+2. Download the [sample files]({{base_path}}/assets/attachments/tutorial/MI_QSG_HOME.zip). From this point onwards, let's refer to this folder as `<MI_QSG_HOME>`.
```
Comment (Mariangela) [inline en/docs/get-started/quick-start-guide/integration-qsg.md:14]: File names should only have lowercase letters and dashes. If there are uppercase, the link will break.

File: en/docs/get-started/quick-start-guide/integration-qsg.md [hunk 2]
```diff
@@ -133,7 +162,7 @@ java -jar DoctorInfo-JDK11.jar
 
 #### Deploy the healthcare service
 
-Copy the CAR file of the healthcare service (HealthcareCompositeExporter_1.0.0.car) from the `<MI_QSG_HOME>/HealthcareCompositeExporter/target/` directory to the `<MI_HOME>/repository/deployment/server/carbonapps` directory.
+Copy the CAR file of the healthcare service (HealthcareIntegrationProjectCompositeExporter_1.0.0-SNAPSHOT.car) from the `<MI_QSG_HOME>/HealthcareIntegrationProject/HealthcareIntegrationProjectCompositeExporter/target/` directory to the `<MI_HOME>/repository/deployment/server/carbonapps` directory.
```
Comment (Mariangela) [inline en/docs/get-started/quick-start-guide/integration-qsg.md:165]: Change file name to lower case

File: en/docs/get-started/quick-start-guide/integration-qsg.md [hunk 3]
```diff
@@ -206,22 +235,49 @@ Upon invocation, you should be able to observe the following response:
...
+
+```toml
+[[service_catalog]]
+apim_host = "https://localhost:9443"
+enable = true
+username = "admin"
+password = "admin"
+```
+    
+Now if you restart Micro Integrator, you can observe that API will be added to the Service Catalog of API Manager.
```
Comment (Mariangela) [inline en/docs/get-started/quick-start-guide/integration-qsg.md:263]: We are not going to mention the word "Service Catalog" as decided by the leads. This should be changed to Services.

Comment (Mariangela) [inline en/docs/get-started/quick-start-guide/integration-qsg.md:263]: "API will be added to the Service Catalog" 

It is the integration service that gets added to the Services page as opposed to the API itself.

File: en/docs/get-started/quick-start-guide/integration-qsg.md [hunk 4]
```diff
@@ -206,22 +235,49 @@ Upon invocation, you should be able to observe the following response:
...
+You will see the following in the server start up log.
+
+```bash
+Successfully updated the service catalog
```
Comment (Mariangela) [inline en/docs/get-started/quick-start-guide/integration-qsg.md:267]: Shouldn't this message change if we are moving away from the term service catalog.

**Review by Mariangela — CHANGES_REQUESTED** [review]

File: en/docs/get-started/quick-start-guide/integration-qsg.md (hunk 1 above)
Comment (arunans23) [reply en/docs/get-started/quick-start-guide/integration-qsg.md]: Fixed

File: en/docs/get-started/quick-start-guide/integration-qsg.md (hunk 2 above)
Comment (arunans23) [reply en/docs/get-started/quick-start-guide/integration-qsg.md]: Fixed

File: en/docs/get-started/quick-start-guide/integration-qsg.md (hunk 3 above)
Comment (arunans23) [reply en/docs/get-started/quick-start-guide/integration-qsg.md]: Fixed

Comment (arunans23) [reply en/docs/get-started/quick-start-guide/integration-qsg.md]: Fixed

File: en/docs/get-started/quick-start-guide/integration-qsg.md (hunk 4 above)
Comment (arunans23) [reply en/docs/get-started/quick-start-guide/integration-qsg.md:267]: This message is already there in the Product. Hence this cannot be changed at the moment

**Review by Mariangela — APPROVED** [review]

**Pull request merged**